src/fal/cli.py

```diff
@@ -50,7 +50,15 @@ def run(run, project_dir, profiles_dir, keyword, all, debug):
 
         project = parse_project(real_project_dir, real_profiles_dir, keyword)
         models = project.get_filtered_models(all)
-        ordered_scripts = ScriptGraph(models, keyword, project_dir).sort()
+
+        models_arr = map(
+            lambda model: f"{model.name}: {', '.join(model.meta['fal']['scripts'])}",
+            models
+        )
+        models_str = '\n'.join(models_arr)
+        print_timestamped_line(f"Starting FAL run for following models (model_name: scripts): \n{models_str}")
+
+        # ordered_scripts = ScriptGraph(models, keyword, project_dir).sort()
```

Comment:
this breaks on a schema.yml with multiple scripts, such as:

```
version: 2

models:
  - name: boston
    description: Ozone levels
    config:
      materialized: table
    meta:
      fal:
        scripts:
          - fal_scripts/slack.py
  - name: stg_zendesk_ticket_data
    description: zendesk ticket data
    config:
      materialized: table
    meta:
      fal:
        scripts:
          - fal_scripts/zendesk_sentiment_analysis.py
  - name: zendesk_ticket_metrics
    description: Zendesk ticket metrics
    config:
      materialized: table
    meta:
      fal:
        scripts:
          - fal_scripts/forecast_slack.py
```

with this error:

```
Exception: Your python scripts contain a cycle could not determine the right order
```

Reply:
@turbo1912